index.html

```diff
@@ -0,0 +1,207 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="CSS/style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="mob-navbar">
+          <a href="#">Edith</a>
+          <img src="assets/download.png" alt="option" />
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="title-section">
+        <div class="landing-text">
+          <h1>Edith Oga</h1>
+          <h2>Software developer</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <ul class="social-media-icons">
+          <li>
+            <a href="#">
+              <img class="icon" src="assets/git.png" alt="Git image" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img class="icon" src="assets/linked.png" alt="Linkedin image" />
+            </a>
+          </li>
+        </ul>
+      </section>
+      <section class="recent-works">
+        <div class="grid-container">
+          <div class="my-work-card">
+            <div class="card-image"></div>
+            <div class="my-works-text">
+              <h2>Multi-post Stories Gain+Glory</h2>
+              <ul class="built-with">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Jav Script</li>
+                <li>html</li>
+              </ul>
+              <input type="button" class="work-button" value="See Project" />
+            </div>
+          </div>
+          <div class="my-work-card">
+            <div class="card-image"></div>
+            <div class="my-works-text">
+              <h2>Multi-post Stories Gain+Glory</h2>
+              <ul class="built-with">
+                <li>Ruby on rails</li>
+                <li>css</li>
+                <li>Jav Script</li>
+                <li>html</li>
+              </ul>
+              <input type="button" class="work-button" value="See Project" />
+            </div>
```

Comment:
- [ ] Kindly make sure your buttons have the correct background colours as seen in the Figma design

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/177016059-974c67e2-205e-4593-86fb-88c45ef54187.png)  |  ![image](https://user-images.githubusercontent.com/74814780/177016066-8a2aafcf-12a9-4355-9915-41fec9ff5b02.png)